<launch>  
  <include file="$(find ros_detectnet_camera)/launch/zed_camera.launch"/>
  <param name="type" value="coco-bottle" /> 
  <param name="ros_topic" value="rgb/image_raw_color" />
  <node name="ros_detectnet_camera_node" pkg="ros_detectnet_camera" type="ros_detectnet_camera_node" output="screen" />
</launch>
